core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -183,32 +254,42 @@ package object config {
     .createOptional
 
   private[spark] val PYSPARK_DRIVER_PYTHON = ConfigBuilder("spark.pyspark.driver.python")
+    .doc("Python binary executable to use for PySpark in driver.")
     .stringConf
     .createOptional
 
   private[spark] val PYSPARK_PYTHON = ConfigBuilder("spark.pyspark.python")
+    .doc("Python binary executable to use for PySpark in both driver and executors.")
     .stringConf
     .createOptional
 
   // To limit memory usage, we only track information for a fixed number of tasks
   private[spark] val UI_RETAINED_TASKS = ConfigBuilder("spark.ui.retainedTasks")
+    .doc("How many tasks the Spark UI and status APIs remember before garbage collecting.")
     .intConf
+    .checkValue(_ > 0, "Number of Tasks event queue must not be negative")
```

Comment:
Since `_ > 0`, we should avoid 0, too.
`must not be negative` => `must be positive`.